djangocms_repeater/templates/multiple-single-header.html

```diff
@@ -1,5 +1,5 @@
 {% load cms_tags %}
-<li class="masthead careers-page" style="background-image:url('{{instance.backgroundImage}}')">
+<li class="masthead careers-page" style="background-image:url('{% firstof instance.background_image.url instance.background_image_url %}')">
```

Comment:
Ah sorry I see these are different they looked exactly the same and looked like it was checking against the same thing. All good!